Mirror/Runtime/NetworkClient.cs

```diff
@@ -277,10 +295,7 @@ void GenerateError(byte error)
 
         public float GetRTT()
```

Comment:
maybe we should make this obsolete with the hint to use NetworkTime.rtt? kinda redundant otherwise